apps/src/templates/census/CensusMap.jsx

```diff
@@ -75,29 +75,31 @@ class CensusMapInfoWindow extends Component {
           <div className={colorClass} />
           {censusMessage}
         </div>
-        <div className="button-container">
-          <div className="button-link-div">
-            <a
-              onClick={() =>
-                this.props.onTakeSurveyClick(schoolDropdownOption, false)
-              }
-            >
-              <div className="button">
-                <div className="button-text">
-                  Take the survey for this school
+        {this.props.onTakeSurveyClick && (
```

Comment:
This is part of the yourschool page so I didn't think this made sense to include on the embedded map.